source/WindowScroller/WindowScroller.css

```diff
@@ -0,0 +1,3 @@
+.disabled > * {
+  pointer-events: none !important;
```

Comment:
If I understand correctly, the intention is to disable pointer events on the contents of whatever element is scrolling, without disabling them on the scrolling element itself. So if the contents can now be _N_ number of nodes, either we use CSS to select all children, or we iterate over all _N_ nodes to add/remove inline styles.

To the initial question, `.disabled > *` and `.disabled *` should have the same effect, and I didn't expect the latter to be more efficient, but that’s fine.

And as far as the name goes, we can definitely change it—since the demo was using CSS Modules, and I'm using CSS Modules in my own stuff, I totally didn't think about it. Another option would be to dynamically generate/insert a stylesheet, but that would be a lot more magic; harder for developers to track down if they have the need to customize behavior.